From df70dbbae4f7ce42ba3296d165363aa160a34113 Mon Sep 17 00:00:00 2001
From: Rico Tzschichholz <ricotz@ubuntu.com>
Date: Wed, 16 Dec 2020 20:19:05 +0100
Subject: [PATCH] gdk: Use correct parameter name to fix g-ir-scanner warning

../gdk/gdktoplevellayout.c:217: Warning: Gdk: gdk_toplevel_layout_get_maximized:
  unknown parameter 'maximized' in documentation comment, should be 'maximize'
---
 gdk/gdktoplevellayout.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gdk/gdktoplevellayout.h b/gdk/gdktoplevellayout.h
index 69aaea8564..827d078968 100644
--- a/gdk/gdktoplevellayout.h
+++ b/gdk/gdktoplevellayout.h
@@ -67,7 +67,7 @@ void                    gdk_toplevel_layout_set_fullscreen (GdkToplevelLayout *l
 
 GDK_AVAILABLE_IN_ALL
 gboolean                gdk_toplevel_layout_get_maximized (GdkToplevelLayout *layout,
-                                                           gboolean          *maximize);
+                                                           gboolean          *maximized);
 
 GDK_AVAILABLE_IN_ALL
 gboolean                gdk_toplevel_layout_get_fullscreen (GdkToplevelLayout *layout,
-- 
GitLab

